src/assets/styles/styles.11ty.js

```diff
@@ -0,0 +1,188 @@
+    async processCss(options) {
+      const from = options.entryPath || 'undefined';
+      const css = postcss([autoprefixer]).process(options.str, {from}).css;
+      return css;
+    }
+
+    // Compile and process CSS (source-agnostic).
+    async compile(options) {
+      let css;
+      switch (options.fileType) {
+        case 'css':
+          css = await this.processCss({str: options.str});
+          break;
+        case 'scss':
+          css = await this.compileScss(options);
+          break;
+        case 'styl':
+          css = await this.compileStylus(options);
+          break;
+      }
+      return css;
+    }
+
+    // Minify & optimize with CleanCSS in production.
+    async minify(css) {
+        return new Promise((resolve, reject) => {
+            if (!isProd) {
+                resolve(css);
+            }
+
+            const minified = new CleanCSS().minify(css);
+            if (minified.styles) {
+                resolve(minified.styles);
+            } else {
+                reject(minified.error);
+            }
+        });
+    }
+
+    // Display an error overlay when CSS build fails.
+    // this brilliant idea is taken from Mike Riethmuller / Supermaya
+    // @see https://github.com/MadeByMike/supermaya/blob/master/site/utils/compile-scss.js
+    renderError(error) {
+        return `
+        /* Error compiling stylesheet */
+        *,
+        *::before,
+        *::after {
+            box-sizing: border-box;
+        }
+        html,
+        body {
+            margin: 0;
+            padding: 0;
+            min-height: 100vh;
+            font-family: monospace;
+            font-size: 1.25rem;
+            line-height:1.5;
+        }
+        body::before {
+            content: '';
+            background: #000;
+            top: 0;
+            bottom: 0;
+            width: 100%;
+            height: 100%;
+            opacity: 0.7;
+            position: fixed;
+        }
+        body::after {
+            content: '${cssesc(error)}';
+            white-space: pre;
+            display: block;
+            top: 0;
+            padding: 30px;
+            margin: 50px;
+            width: calc(100% - 100px);
+            color:#721c24;
+            background: #f8d7da;
+            border: solid 2px red;
+            position: fixed;
+        }`;
+  }
+
+    // Render the CSS file.
+    async render({entryPath}) {
+        try {
+            const fileType = entryPath.split('.').pop();
+            const str = fs.readFileSync(entryPath, 'utf8');
+            const css = await this.compile({fileType, entryPath, str});
+            const result = await this.minify(css);
+            return result;
+        } catch (error) {
+            if (isProd) {
+                // Throw and bail if in a production environment.
+                throw new Error(error);
+            } else {
+                // Log and display the error.
+                console.error(error);
```

Comment:
If you are using the eslint-config-fluid this should get flagged. You'll need ignore the rule for this line if you want the logging to happen.

Reply:
The linting fails and stops at line 33 because of the `async` keyword. I'll need to update to the newer version.